src/workloads/sharding/ReshardCollectionWithIndexes.yml

```diff
@@ -0,0 +1,174 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/sharding"
+Description: |
+  This test expects the target cluster is created using ebs snapshot with 1 billion 1KB records 
+  and has 3 shards, named shard-00, shard-01, shard-02. The collection has 10 indexes including 
+  _id index, single-key index and compound index. The whole dataset is on 1 shard at the beginning.
+  ReshardCollection should use same-key resharding and use shardDistribution parameter to reshard 
+  into 2 shards then 3 shards. There will be random CRUD operations during resharding but should 
+  run at a very low rate.
```

Comment:
Is the important part that there is something running in the background, but what it is is not very important? Should you be driving conflicts in which crud operations target actively being moved data?

Reply:
The random CRUD operations here are to make sure that these operations work as normal and won't make resharding fail, but as the goal of this test is to see the performance improvements of resharding, I can also remove these operations.

Reply:
Don't remove the background operations for me. I'm just pushing for clarity and making sure things are intentional. 

Do you expect any impact on the background load? It might be worth explicitly saying that the performance of the background crud operations is of secondary concern to the performance of the resharding operation in this test (assuming those words are true).